tests/e2e/default/more-options-menu/online-user.wdio-spec.js

```diff
@@ -0,0 +1,119 @@
+const health_center = places.get('health_center');
+const district_hospital = places.get('district_hospital');
+
+const contact = personFactory.build({
+  _id: uuid(),
+  name: 'contact',
+  phone: '+12068881234',
+  place: health_center._id,
+  type: 'person',
+  parent: {
+    _id: health_center._id,
+    parent: health_center.parent
+  },
+});
+
+const patient = personFactory.build({
+  _id: uuid(),
+  parent: { _id: clinic._id, parent: { _id: health_center._id, parent: { _id: district_hospital._id }}}
+});
+
+const report = reportFactory.build({ form: 'home_visit', content_type: 'xml' }, { patient, submitter: contact });
+
+const sendMessage = async (message = 'Testing', phone = contact.phone) => {
+  await utils.request({
+    method: 'POST',
+    path: '/api/v2/records',
+    headers: {
+      'Content-type': 'application/x-www-form-urlencoded'
+    },
+    body:`message=${message}&from=${phone}`,
+  });  
+};
+
+describe('Online User', async () => {  
+  before(async () => {
+    await loginPage.cookieLogin();
+  });
+  describe('Options disabled when no items: messages, contacts, people', async () => {
+    it('- Message tab', async () => {
+      await commonPage.goToMessages();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'messages')).to.be.false;    
+    });
+
+    it(' - Contact tab', async () => {
+      await commonPage.goToPeople();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'contacts')).to.be.false;
+      expect(await commonPage.isOptionVisible('edit', 'contacts')).to.be.false;
+      expect(await commonPage.isOptionVisible('delete', 'contacts')).to.be.false;     
+    });
+
+    it('- Report tab', async () => {
+      await commonPage.goToReports();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'reports')).to.be.false;
+      expect(await commonPage.isOptionVisible('edit', 'reports')).to.be.false;
+      expect(await commonPage.isOptionVisible('delete', 'reports')).to.be.false;     
+    });
+  });
+
+  describe(' - Options enabled when there are items: messages, contacts, peope', async () => {
+    before(async () => {
+      await utils.saveDocs([ ...places.values(), contact, patient, report ]);
+      await sendMessage();    
+    });
+
+    it('- Contact tab: no contact selected', async () => {
+      await commonPage.goToPeople();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'contacts')).to.be.true;
+      expect(await commonPage.isOptionVisible('edit', 'contacts')).to.be.false;
+      expect(await commonPage.isOptionVisible('delete', 'contacts')).to.be.false;
+    });
+
+    it(' - Contact Tab : contact selected', async () => {
+      await commonPage.goToPeople();
+      await contactPage.selectLHSRowByText(contact.name);
+      await (await contactPage.contentRow()).waitForDisplayed();
```

Comment:
Do we need to do the same (line 91, 92, 93) in the other tests to avoid flaky? 🤔